tests/hpc/package_check.pm

```diff
@@ -0,0 +1,139 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: HPC package(s) checks
+#  The test is meant as rudimentary check of some white-listed specific
+#  packages. The white list should be located in data folder and should
+#  contain the list of rpms with expected minimal versions. This white
+#  list is then used for basic checks if packages are installable, if
+#  the version is as expected and then single basic tests is run for
+#  each package.
+# Maintainer: Sebastian Chlad <sebastian.chlad@suse.com>
+
+use base 'hpcbase';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub check_version {
+    my ($rpm, $expected_version) = @_;
+
+    my $installed_rpm_ver;
+
+    if ($rpm =~ s/papi//) {
+        $installed_rpm_ver = script_output("rpm -qa | grep papi");
+        $installed_rpm_ver =~ s/papi-//;
+	$installed_rpm_ver =~ s/[.]//g;
+	$installed_rpm_ver = substr($installed_rpm_ver, 0, 3);
+    } else {
+        $installed_rpm_ver = script_output("$rpm -v");
+        $installed_rpm_ver =~ s/$rpm version//;
+        $installed_rpm_ver =~ s{^\s+|\s+$}{}g;
+    }
+
+    ## make casting explicit
+    my $num1 = int($expected_version);
```

Comment:
I think we run in problems with version like `5.7.0`. For instance `perl -e 'print int("5.7.0")'` return `5`, I guess this is not what we want.
Lets use something like https://metacpan.org/pod/Sort::Versions, it is already a requirement from os-autoinst/openQA.

Reply:
this is awesome pointer. Sort::Versions simply works. Thx!